openverse_catalog/dags/providers/provider_workflow_types.py

```diff
@@ -0,0 +1,167 @@
+from dataclasses import dataclass
```

Comment:
This and the ingestion workflow `_types.py` file both end as such but include more than types (since they also include the workflow definitions themselves). Should we rename this to just `provider_workflows.py` (and similarly for ingestion)?